articles/multifactor-authentication/index2.md

```diff
@@ -15,49 +15,45 @@ Multifactor Authentication (MFA) is a method of verifying a user's identity by r
 
 Auth0 supports the following methods of implementing MFA:
 
-1. Using Guardian, Auth0's MFA application;
-2. Using Auth0's built-in support for one-time password authentication services Google Authenticator and Duo Security.
-3. Configuring rules for custom processes, such as Contextual MFA, which allow you to define the conditions that will trigger additional authentication challenges, such as changes in geographic location or logins from unrecognized devices.
-4. Integration with a custom provider, such as **Yubikey**.
+1. [Guardian](/multifactor-authentication2#mfa-using-auth0-guardian)- Auth0's application for MFA
+2. [SMS](/multifactor-authentication2#mfa-with-sms)- Verification by sending a six-digit code via SMS
+3. Support for one-time password authentication services [Google Authenticator](/multifactor-authentication2#mfa-using-google-authenticator) and [Duo Security](/multifactor-authentication2#mfa-using-duo-security).
+4. [Configuring rules for custom processes](/multifactor-authentication2#mfa-using-custom-rules)- such as Contextual MFA, which allow you to define the conditions that will trigger additional authentication challenges, such as changes in geographic location or logins from unrecognized devices.
+5. Using a [custom provider](/multifactor-authentication2#mfa-using-a-custom-provider), such as **Yubikey**.
 
-### MFA Using Auth0 Guardian
+## MFA Using Auth0 Guardian 
 
 Guardian is Auth0's MFA application. It is a frictionless approach to implementing MFA for your apps, and provides a full MFA experience without requiring integration with third-party utilities.
 
-You will find further instructions on enabling this feature [here](/multifactor-authentication/guardian/configuration.md).
+[Click here to learn more about enabling Guardian](/multifactor-authentication/guardian)
 
-### MFA Using Auth0 Built-In MFA Support
+## MFA with SMS
 
-Auth0 provides built-in support for the following one-time password authentication services:
+Auth0 supports sending an SMS with a one time password code to be used for another step of verification.
```

Comment:
a minor detail, not sure what style is preferred `one-time` vs `one time` but we might want to normalize the style and choose one just for consistency.